libnm-core/nm-utils.c

```diff
@@ -6223,6 +6223,33 @@ nm_utils_format_variant_attributes (GHashTable *attributes,
 
 /*****************************************************************************/
 
+/*
+ * nm_utils_get_timestamp_msec():
+ *
+ * Gets current time in milliseconds of CLOCK_BOOTTIME.
```

Comment:
That this is CLOCK_BOOTTIME is an implementation detail. It's not even necessary to document it (but fine, if you want).

More important however is, that the returned time is the same source and scale, as various timestamps that are exposed by NetworkManager on D-Bus.

Also say, that the value only has meaning on the same host, and that it is increasing monotonically.